lib/pages/dashboard_page.dart

```diff
@@ -24,108 +26,120 @@ class _DashboardPageState extends State<DashboardPage> {
-                          selectedItemList[2] = false;
-                          selectedItemList[3] = true;
-                        });
-                      },
-                      child: listItem(
-                          context, 'Report', constraints, selectedItemList[3]),
-                    ),
-                  ],
+                        ],
+                      ),
+                      SizedBox(
+                        height: 16,
+                      ),
+                      InkWell(
+                        onTap: () {
+                          setState(() {
+                            selectedItem = 0;
+                            selectedItemList[0] = true;
+                            selectedItemList[1] = false;
+                            selectedItemList[2] = false;
+                            selectedItemList[3] = false;
+                          });
+                        },
+                        child: listItem(
+                            context, 'Items', constraints, selectedItemList[0]),
+                      ),
+                      SizedBox(
+                        height: 8,
+                      ),
+                      InkWell(
+                        onTap: () {
+                          setState(() {
+                            selectedItem = 1;
+                            selectedItemList[0] = false;
+                            selectedItemList[1] = true;
+                            selectedItemList[2] = false;
+                            selectedItemList[3] = false;
+                          });
+                        },
+                        child: listItem(context, 'Purchase', constraints,
+                            selectedItemList[1]),
+                      ),
+                      SizedBox(
+                        height: 8,
+                      ),
+                      InkWell(
+                        onTap: () {
+                          setState(() {
+                            selectedItem = 2;
+                            selectedItemList[0] = false;
+                            selectedItemList[1] = false;
+                            selectedItemList[2] = true;
+                            selectedItemList[3] = false;
+                          });
+                        },
+                        child: listItem(
+                            context, 'Sales', constraints, selectedItemList[2]),
+                      ),
+                      SizedBox(
+                        height: 8,
+                      ),
+                      InkWell(
+                        onTap: () {
+                          setState(() {
+                            selectedItem = 3;
+                            selectedItemList[0] = false;
+                            selectedItemList[1] = false;
+                            selectedItemList[2] = false;
+                            selectedItemList[3] = true;
```

Comment:
@ahamzatariq , Please fix this in the initial PR and backmerge the code all the way here.

Reply:
Fixed. But didn't backmerge.